.github/workflows/run.yaml

```diff
@@ -0,0 +1,89 @@
+---
+name: 'Drupal 9 updates'
+
+on:
+  push:
+    branches-ignore:
+      - 'master'
```

Comment:
should this be main? or shall we comment this off too?

Reply:
Copy pasta, definitely should be.